doc/manual/50-feature-details.en.md

```diff
@@ -238,20 +238,35 @@ rules:
 > From Barman 2.10, you can use the `--wait` option with `barman backup`
 > command.
 
-#### Current limitations on backup from standby
+### Concurrent backup of a standby
 
-Barman currently requires that WAL files and backup data come from the
-same PostgreSQL server. When taking backups from a standby it is therefore
-important to point the following Barman [configuration options][config-options]
-to the standby server:
+If backing up a standby then the following [configuration options][config-options]
+should point to the standby server:
 
 - `conninfo`
 - `streaming_conninfo` (when using `backup_method = postgres` or `streaming_archiver = on`)
 - `ssh_command` (when using `backup_method = rsync`)
 
-In the case that the standby is promoted to primary the backups and WALs will
-continue to be valid however you may wish to update the Barman configuration
-so that it uses the new standby for taking backups and receiving WALs.
+The following config option should point to the primary server:
+
+- `primary_conninfo`
+
+Barman will use `primary_conninfo` to switch to a new WAL on the primary
+so that the concurrent backup against the standby can complete without
+having to wait for a WAL switch to occur naturally.
+
+> **NOTE:** It is especially important that `primary_conninfo` is
+> set if the standby is to be backed up when there is little or no write
+> traffic on the primary. If `primary_conninfo` is not set then the
+> backup will still run however it will wait at the stop backup stage
+> until the current WAL semgent on the primary is newer than the latest
+> WAL required by the backup.
+
+Barman currently requires that WAL files and backup data come from the
+same PostgreSQL server. In the case that the standby is promoted to primary
+the backups and WALs will continue to be valid however you may wish to update
```

Comment:
Yes we'd need to either remove `primary_conninfo` or point it to the new primary. If nothing is updated after the standby is promoted then `barman check` is going to fail because of [Server._check_standby](https://github.com/EnterpriseDB/barman/pull/612/files#diff-40d894681de880b0b65b0db09b8c805d98a794a302d7fdcdb82e205f3952a12fR861) so backups aren't going to run until it is updated.